From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Wed, 29 May 2024 11:34:40 +0800
Subject: Patching u-boot rk35xx files arch/arm/dts/rk3588s-nanopi-r6c.dts

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 arch/arm/dts/rk3588s-nanopi-r6c.dts | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c.dts b/arch/arm/dts/rk3588s-nanopi-r6c.dts
index b7bca848ac..93e3c42702 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c.dts
+++ b/arch/arm/dts/rk3588s-nanopi-r6c.dts
@@ -5,15 +5,10 @@
 
 / {
 	model = "RK3588S NanoPi R6C";
 	compatible = "rockchip,rk3588s-nanopi-r6c", "rockchip,rk3588s";
 
-	chosen: chosen {
-		stdout-path = &uart2;
-		u-boot,spl-boot-order = &sdmmc, &sdhci;
-	};
-
 	aliases: aliases {
 		mmc0 = &sdhci;
 		mmc1 = &sdmmc;
 	};
 
@@ -195,6 +190,5 @@
 };
 
 &uart5 {
 	status = "okay";
 };
-
-- 
Created with Armbian build tools https://github.com/armbian/build

