From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Wed, 29 May 2024 12:16:27 +0800
Subject: Patching u-boot rk35xx files arch/arm/dts/rk3588s-nanopi-r6c.dts

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 arch/arm/dts/rk3588s-nanopi-r6c.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c.dts b/arch/arm/dts/rk3588s-nanopi-r6c.dts
index d603e21b29..4c16d24e73 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c.dts
+++ b/arch/arm/dts/rk3588s-nanopi-r6c.dts
@@ -5,10 +5,15 @@
 
 / {
 	model = "RK3588S NanoPi R6C";
 	compatible = "rockchip,rk3588s-nanopi-r6c", "rockchip,rk3588s";
 
+	chosen: chosen {
+		stdout-path = &uart5;
+		u-boot,spl-boot-order = &sdmmc, &sdhci;
+	};
+
 	aliases: aliases {
 		mmc0 = &sdhci;
 		mmc1 = &sdmmc;
 	};
 
-- 
Created with Armbian build tools https://github.com/armbian/build

