From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Thu, 30 May 2024 17:13:45 +0800
Subject: Patching u-boot rk35xx files arch/arm/dts/rk3588s-nanopi-r6c.dts

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 arch/arm/dts/rk3588s-nanopi-r6c.dts | 18 ++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c.dts b/arch/arm/dts/rk3588s-nanopi-r6c.dts
index 3dccdbc6d7..9debd7cbd0 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c.dts
+++ b/arch/arm/dts/rk3588s-nanopi-r6c.dts
@@ -5,15 +5,33 @@
 
 / {
 	model = "RK3588S NanoPi R6C";
 	compatible = "rockchip,rk3588s-nanopi-r6c", "rockchip,rk3588s";
 
+	chosen: chosen {
+		// stdout-path = &uart2;
+		u-boot,spl-boot-order = &sdmmc, &sdhci;
+	};
+
+	aliases: aliases {
+		mmc0 = &sdhci;
+		mmc1 = &sdmmc;
+	};
+
 	vcc5v0_sys: vcc5v0-sys {
 		u-boot,dm-pre-reloc;
 		compatible = "regulator-fixed";
 		regulator-name = "vcc5v0_sys";
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
 		regulator-always-on;
 		regulator-boot-on;
 	};
 };
+
+// &crypto {
+// 	status = "okay";
+// };
+
+// &uart2 {
+// 	status = "okay";
+// };
\ No newline at end of file
-- 
Created with Armbian build tools https://github.com/armbian/build

